Makefile

```diff
@@ -7,11 +7,12 @@ dist: $(DISTFILE)
 install:
 	@rm -rf $(DESTDIR)
 	./tools/install.py $(DESTDIR)
+	npm install --prefix $(DESTDIR)
 
 # Create an distributable archive of emscripten suitable for use
 # by end users.  This archive excludes parts of the codebase that
 # are you only used by emscripten developers.
 $(DISTFILE): install
-	tar cf $@ $(EXCLUDE_PATTERN) -C `dirname $(DESTDIR)` `basename $(DESTDIR)`
+	tar cf $@ $(EXCLUDE_PATTERN) --exclude=node_modules -C `dirname $(DESTDIR)` `basename $(DESTDIR)`
```

Comment:
I think this is what `EXCLUDE_PATTERN` used to be for.  Can you use (or just remove it?)?

Reply:
I didn't find a way for EXCLUDE_PATTERN to be used actually and assumed it was some magic I didn't understand. I'll be happy to take it out with your approval, though.

Reply:
This file used to to have `EXCLUDE_PATTERN=--exclude='*.pyc' --exclude='*/__pycache__'` up top before #10837.

Reply:
In that case, it should be fine now. :)